src/state/store.ts

```diff
@@ -30,12 +30,14 @@ interface Store {
   readonly alias: string | undefined;
   readonly balance: number | undefined;
   readonly connectorName: string | undefined;
+  readonly appName: string | undefined;
 
   connect(config: ConnectorConfig): void;
   disconnect(): void;
   setAlias(alias: string | undefined): void;
   setBalance(balance: number | undefined): void;
   setRoute(route: Route): void;
+  setAppName(appName: string | undefined): void;
```

Comment:
this should only accept a string, not undefined

Reply:
I guess the other setters should also not accept undefined. I will fix that in  a separate PR :o